doc/manuals/user/forbidden_characters.md

```diff
@@ -59,3 +59,42 @@ NGSIv2 introduces syntax restrictions for ID fields (such as entity id/type, att
 or metadata name/type) which are described in the "Field syntax restrictions" section in the
 [NGSIv2 specification](http://telefonicaid.github.io/fiware-orion/api/v2/). You can also
 enable them for NGSIv1, as described in [this section of the documentation](v1_v2_coexistence.md#checking-id-fields).
+
+### Notification payload template
+
+NGSIv2 allows a templating mechanism (see "Notification templates" section in
```

Comment:
Note that the special case of URL decoding in payload template is Orion implementation specific (a consecuence of forbidden chars in Orion) so it is not described in the .apib but in the .md. If you think that there is a better place than as sub-section in forbidden_characters.md, please suggest it.